packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.scss

```diff
@@ -217,6 +217,16 @@
   }
 }
 
+.gux-avatar-integration-badge {
+  position: absolute;
+  inset-block-end: 0;
+  inset-inline-end: 0;
+  z-index: 2;
```

Comment:
Needed to bring the logo above the overlay in the gux-avatar-change-photo version. Had a look at the z-index tokens but the meaning seemed a little different so wasn't really sure which suited this purpose.

Reply:
The semantic tokens are not useful here plus you have created a new stacking context so no semantic token would make sense. You could use --gse-core-zIndex-1 for a value of '1' (I'm not sure why you have '2').

Anyway...it doesn't seem to do anything, is it needed at all?

Reply:
It's basically to account for the situation where it's used in combination with gux-avatar-change-photo. In that case, the overlay has a z-index of 1 to pop it above the avatar. Then the app logo will be partially hidden if it has the same z-index as the rest of the avatar. 

`  <gux-avatar-focusable-beta>
    <gux-avatar-change-photo-beta>
      <gux-avatar-beta
        slot="avatar"
        name="Conor Darcy"
        presence="available"
        badge
        ring
      ></gux-avatar-beta>
    </gux-avatar-change-photo-beta>
  </gux-avatar-focusable-beta>`

Reply:
Ok, gotcha. Maybe you could use the same token you have for the badge? I'm sure you can have multiple elements on the same level.

Reply:
Of course, should've copped that. Updated now